modmesh/pilot/_gui.py

```diff
@@ -128,10 +128,17 @@ def _addAction(menu, text, tip, func):
             )
 
         _addAction(
-            menu=wm.addonMenu,
+            menu=wm.oneMenu,
             text="(To be deprecated) load linear_wave",
             tip="Load linear_wave",
-            func="modmesh.app.linear_wave.load_app",
+            func="modmesh.onedim.linear_wave.load_app",
+        )
+
+        _addAction(
+            menu=wm.addonMenu,
```

Comment:
Would it be good to remove the addon menu since it is empty?

Reply:
I kept the Addon menu for the upcoming plugin system, but I agree that removing it now would simplify the UI. We can reintroduce it later when the new system is ready.

Reply:
We are aligned.  It'd be good to remove the addon menu now and add it later.  But I am OK to leave it now, if you like.